NovaCustom-PD50-20231126-144950.yml

```diff
@@ -0,0 +1,60 @@
+layout:
+ ‘hcl’
+type:
+ ‘laptop’
+hvm:
+ ‘yes’
+iommu:
+ ‘yes’
+slat:
+ ‘yes’
+tpm:
+ ‘2.0’
+remap:
+ ‘yes’
+brand: |
+ NovaCustom
+model: |
+ PD50
+bios: |
+ 1.07.07TDES
+cpu: |
+ 13th Gen Intel(R) Core™ i7-13700HX
+cpu-short: |
+ i7-13700HX
+chipset: |
+ Intel Corporation Device [8086:4637] (rev 02)
+chipset-short: |
+ Raptor Lake
+gpu: |
+ Intel Corporation Alder Lake-HX GT1 [UHD Graphics 770] [8086:4688] (rev 0c) (prog-if 00 [VGA controller])
+ NVIDIA Corporation AD107M [GeForce RTX 4060 Max-Q / Mobile] [10de:28e0] (rev a1) (prog-if 00 [VGA controller])
+gpu-short: |
+ Alder Lake & RTX 4060 Mobile
```

Comment:
This should read `Integrated Graphics (UHD 770) & RTX 4060 Mobile` to be consistent with existing reports.